templates/components/cart/content.html

```diff
@@ -140,13 +140,13 @@ <h2 class="cart-item-name">
 
                 <td class="cart-item-block cart-item-info">
                     <span class="cart-item-label">{{lang 'cart.checkout.total'}}</span>
-                    {{#or ../customer (if ../theme_settings.restrict_to_login '!==' true)}}
+                    {{#or ../customer (unless ../settings.hide_price_from_guests) (if type '==' 'GiftCertificate')}}
                         <strong class="cart-item-value {{#if total_discounted}}price--discounted{{/if}}">{{total.formatted}}</strong>
                         {{#if total_discounted}}
                             <strong class="cart-item-value">{{total_discounted.formatted}}</strong>
                         {{/if}}
                     {{else}}
-                        {{> components/common/login-for-pricing}}
```

Comment:
@bc-yevhenii-buliuk This is because we are already displaying the **login for pricing** message under the **Price** column, so adding it underneath the **Total** column would look redundant and unnecessary. Here's how it looks currently:
![image](https://user-images.githubusercontent.com/19815878/190700184-f8247460-5899-4f3e-b478-36ee16bbac0c.png)

If we left the message in the total column, it would look like this (seems unnecessary):
![image](https://user-images.githubusercontent.com/19815878/190700310-43fb8330-0d2a-446d-b462-c6ec67a68a3e.png)

Reply:
@jordanarldt If it suits the design, then no problem!

Reply:
@bc-yevhenii-buliuk perfect! Resolving this